main.js

```diff
@@ -0,0 +1,308 @@
+let player = "X"
+let startButton = document.getElementById('start');
+let status = document.getElementById('status');
+
+
+let row1 = document.getElementById('row1');
+let cell1 = document.getElementById('cell1');
+let cell2 = document.getElementById('cell2');
+let cell3 = document.getElementById('cell3');
+
+let row2 = document.getElementById('row2');
+let cell4 = document.getElementById('cell4');
+let cell5 = document.getElementById('cell5');
+let cell6 = document.getElementById('cell6');
+
+let row3 = document.getElementById('row3');
+let cell7 = document.getElementById('cell7');
+let cell8 = document.getElementById('cell8');
+let cell9 = document.getElementById('cell9');
+winConditions()
+function winConditions() {
+    // Check if X wins
+    if (cell1 === 'X' && cell2 === 'X' && cell3 === 'X') Winner = 'X';
+    if (cell4 === 'X' && cell5 === 'X' && cell6 === 'X') Winner = 'X';
+    if (cell7 === 'X' && cell8 === 'X' && cell9 === 'X') Winner = 'X';
+    if (cell1 === 'X' && cell4 === 'X' && cell7 === 'X') Winner = 'X';
+    if (cell2 === 'X' && cell5 === 'X' && cell8 === 'X') Winner = 'X';
+    if (cell3 === 'X' && cell6 === 'X' && cell9 === 'X') Winner = 'X';
+    if (cell1 === 'X' && cell5 === 'X' && cell9 === 'X') Winner = 'X';
+    if (cell3 === 'X' && cell5 === 'X' && cell7 === 'X') Winner = 'X';
+    if (Winner === 'X') {
+        console.log("Player X wins the game.");
+
+        // Check if O wins
+        if (cell1 === 'O' && cell2 === 'O' && cell3 === 'O') Winner = 'O';
+        if (cell4 === 'O' && cell5 === 'O' && cell6 === 'O') Winner = 'O';
+        if (cell7 === 'O' && cell8 === 'O' && cell9 === 'O') Winner = 'O';
+        if (cell1 === 'O' && cell4 === 'O' && cell7 === 'O') Winner = 'O';
+        if (cell2 === 'O' && cell5 === 'O' && cell8 === 'O') Winner = 'O';
+        if (cell3 === 'O' && cell6 === 'O' && cell9 === 'O') Winner = 'O';
+        if (cell1 === 'O' && cell5 === 'O' && cell9 === 'O') Winner = 'O';
+        if (cell3 === 'O' && cell5 === 'O' && cell7 === 'O') Winner = 'O';
+        if (Winner === 'O') {
+            console.log("Player2 wins the game.");
+
+        }
+    }
+}
+
+
+
+
+
+start()
+function start() {
+    for (let i = 1; i < 10; i++) {
+        let startSquare = document.getElementById(`cell-${i}`)
+        startSquare.addEventListener('click', play);
+    }
+}
+function clearBoard(){
+    console.log('');
+    console.log('----- New Game -----');
+    winner = null;
+    lineDiv.innerHTML = '';
+    player = letterX
+    playerXMoves = [];
+    playerOMoves = [];
+    currentArray = playerXMoves;
+    playerTurn.innerHTML = "It is player X's turn";
+    turnCount = 0;
```

Comment:
This code is going to cause some bugs since it is using some things that aren't available in your code such as the `letterX` variable and the `lineDiv` element. I don't mind people looking at my code (or someone else's code for that matter) for inspiration, but rather than copying it over wholesale read through it and try to extract only the parts you need for your code.